code/src/components/Form.js

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+
+const Form = ({ onFormSubmit, onNewThought, onNewThoughtChange, charactersCount }) => {
+  return (
+    <form onSubmit={onFormSubmit}>
+      <h1>Whats making you happy right now?</h1>
+      <div className="text-input">
+        <textarea className={charactersCount > 140 ? 'disabled-textarea' : 'textarea'} name="thought" value={onNewThought} onChange={onNewThoughtChange} placeholder="Write here...">lalalallala</textarea>
```

Comment:
Interesting feature that you made it switch between disabled and not disabled textarea! Didn't understand that at first (where the "lalalala"-text goes), so maybe att a comment?